src/open_dread_rando/misc_patches/model_patcher.py

```diff
@@ -16,6 +17,52 @@ class ModelData:
     # dictionary connecting model names (i.e. "mp_opaque_01") to material files
     materials: typing.Optional[dict[str, str]] = None
 
+# contains static data to generate model/material changes
+@dataclasses.dataclass()
+class StaticModelChanger:
+    base_model_path: str
+    new_model_path: str
+    materials: dict[str, MaterialData] # key is material name in bcmdl
+
+    def generate(self, editor: PatcherEditor):
+        mats: dict[str, str] = {}
+
+        for name, mat in self.materials.items():
+            create_custom_material(editor, mat)
+            mats[name] = mat.new_path
+
+        print(self.base_model_path)
```

Comment:
Looks like it was for debugging?
Remove it or in case you want to keep it, use the logger.